sign/sign.go

```diff
@@ -45,7 +46,7 @@ func UnmarshalTransaction(bz []byte) (*Transaction, error) {
 	dec := json.NewDecoder(bytes.NewBuffer(bz))
 	dec.DisallowUnknownFields()
 
-	if err := dec.Decode(s); err != nil {
+	if err := eris.Wrap(dec.Decode(s), ""); err != nil {
 		return nil, fmt.Errorf("error decoding Transaction: %w", err)
```

Comment:
it would be more natural to do:

```go
	if err := dec.Decode(s); err != nil {
		return nil, eris.Wrap(err, "error decoding Transaction")
	}
```

Reply:
@technicallyty I changed it. I also took a closer look at the readme. 

They do this because some functions return both a result and an error and eris can't wrap it on one line so they wrap it in the return value. 

For functions that only return an error we should wrap it on the same line which is what's going on in my code here.

The eris.Wrap function is explicitly designed to be used this way as it returns a nil if the input is nil. 

We can address this issue in another PR.